filestack/src/main/java/com/filestack/android/internal/LocalFilesFragment.java

```diff
@@ -70,7 +73,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle st
         ImageViewCompat.setImageTintList((uploadLocalFilesImageView), ColorStateList.valueOf(theme.getTextColor()));
         RecyclerView filesListView = view.findViewById(R.id.gallery_list);
         filesListView.setLayoutManager(new LinearLayoutManager(getContext()));
-        filesListView.setItemAnimator(new DefaultItemAnimator());
+        filesListView.setItemAnimator(null);
```

Comment:
Specific [requirement](https://fieldwire.atlassian.net/browse/ME-231?focusedCommentId=19852) to remove animation